### PR TITLE
chore(server): update swagger info for default server profile

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
## Purpose of the PR

- close #xxx  

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/profile/ProfileAPI.java [hunk 1]
```diff
@@ -55,6 +55,7 @@ public class ProfileAPI {
     private static final String SERVICE = "hugegraph";
     private static final String DOC = "https://hugegraph.apache.org/docs/";
     private static final String API_DOC = DOC + "clients/";
+    private static final String SWAGGER_UI = "http://{domain/ip}:{port}/swagger-ui/index.html#/";
```
Comment (imbajin) [inline hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/profile/ProfileAPI.java:58]: Shall we need/keep `#` in the end? (whats for?_

**Review by imbajin — APPROVED** [review]

**Review by VGalaxies — COMMENTED** [review]
@SunnyBoy-WYH Is this as expected?

```bash
> curl http://localhost:8080/
{"service":"hugegraph","version":"1.2.0","doc":"https://hugegraph.apache.org/docs/","api_doc":"https://hugegraph.apache.org/docs/clients/","swagger_ui":"http://{domain/ip}:{port}/swagger-ui/index.html#/","apis":["arthas","auth","cypher","filter","graph","gremlin","job","metrics","profile","raft","resources","schema","traversers","variables"]}
```

Is there a way to fill the ip and port in swagger_ui here?

**Comment by imbajin** [comment]
> @SunnyBoy-WYH Is this as expected?
> 
> ```shell
> > curl http://localhost:8080/
> {"service":"hugegraph","version":"1.2.0","doc":"https://hugegraph.apache.org/docs/","api_doc":"https://hugegraph.apache.org/docs/clients/","swagger_ui":"http://{domain/ip}:{port}/swagger-ui/index.html#/","apis":["arthas","auth","cypher","filter","graph","gremlin","job","metrics","profile","raft","resources","schema","traversers","variables"]}
> ```
> 
> Is there a way to fill the ip and port in swagger_ui here?

Good suggestion, maybe it's better to get the `uri info` from server status or use default `IP:Port` for it?

**Comment by SunnyBoy-WYH** [comment]
> > @SunnyBoy-WYH Is this as expected?
> > ```shell
> > > curl http://localhost:8080/
> > {"service":"hugegraph","version":"1.2.0","doc":"https://hugegraph.apache.org/docs/","api_doc":"https://hugegraph.apache.org/docs/clients/","swagger_ui":"http://{domain/ip}:{port}/swagger-ui/index.html#/","apis":["arthas","auth","cypher","filter","graph","gremlin","job","metrics","profile","raft","resources","schema","traversers","variables"]}
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Is there a way to fill the ip and port in swagger_ui here?
> 
> Good suggestion, maybe it's better to get the `uri info` from server status or use default `IP:Port` for it?

Yes, we can just use rest server url .

**Review by imbajin — APPROVED** [review]
good

**Review by VGalaxies — APPROVED** [review]
LGTM~

**Pull request merged**